vendor.mod

```diff
@@ -203,3 +201,5 @@ require (
 	k8s.io/klog/v2 v2.80.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/moby/buildkit => github.com/jedevc/buildkit v0.8.2-0.20230711123235-03840b20c83a
```

Comment:
Also needs to rebase your for with latest v0.11 changes

Reply:
Done, let's see how this goes :crossed_fingers:

Reply:
Needs another batch of rebases :see_no_evil: